### PR TITLE
Update README for v0.5.0

### PR DESCRIPTION
**Pull request by achambers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,90 @@
...
+if (context.revisionKey && !context.activatedRevisionKey) {
+  return "Deployed but did not activate revision " + context.revisionKey + ". "
+       + "To activate, run: "
+       + "ember activate " + context.revisionKey + " --environment=" + context.deployEnvironment + "\n";
+}
+```
 
-For more information on using ember-cli, visit [http://www.ember-cli.com/](http://www.ember-cli.com/).
+[1]: https://github.com/lukemelia/ember-cli-deploy-lightning-pack "ember-cli-deploy-lightening-pack"
+[2]: https://ember-cli-deploy.com/something "Plugin Documentation"
```
Comment (achambers) [inline README.md]: Need to make sure there is somewhere for this link to go

File: README.md [hunk 2]
```diff
@@ -1,25 +1,109 @@
 # Ember-cli-deploy-redis
 
-This README outlines the details of collaborating on this Ember addon.
+> An ember-cli-deploy-plugin to upload files to a Redis store
+
+**This plugin is only compatible with ember-cli-deploy versions >= 0.5.0**\
```
Comment (lukemelia) [inline README.md]: Remove extra slash

File: README.md [hunk 3]
```diff
@@ -1,25 +1,109 @@
...
+
+This plugin uploads one or more files to a specified Redis store.
+
+More often than not this plugin will be used in conjunction with the [lightening method of deployment][1] where the ember application assets will be served from S3 and the index.html file will be served from Redis. However, it can be used to upload any files to a Redis store.
```
Comment (lukemelia) [inline README.md]: should be "lightning" http://grammarist.com/spelling/lightening-lightning/

File: README.md [hunk 4]
```diff
@@ -1,25 +1,109 @@
...
+
+*Example:* 'redis://some-user:some-password@some-host.com:1234'
+
+### filePattern
+
+Files matching this pattern will be uploaded to Redis. By default this option will use the `distFiles` property of the deployment context.
 
-* `ember test`
-* `ember test --server`
+*Default:* `context.distFiles + '/index.html'`
```
Comment (lukemelia) [inline README.md]: This should probably be `distDir`, but what's more, this value should probably just be `"index.html"` and be applied to files in `distDir`, which would be it's own configuration option. Is there a use case for deploying something that is not under `distDir`?

**Comment by lukemelia** [comment]
This is looking good. We may want to have an example configuration section to give people something to copy and paste.

**Comment by lukemelia** [comment]
Perhaps "Minimal config" and "Example config"

File: README.md (hunk 4 above)
Comment (achambers) [inline README.md]: You're probably right about that actually.  I'll make the change to split out the pattern and the distFiles. As for deploying files outside of distDir, I haven't come across a use case myself.  Let's see how that plays out.

**Pull request merged**